deploy/9030_add_gems_and_catalysts.ts

```diff
@@ -37,7 +37,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   await execute(
     'GemsCatalystsRegistry',
-    {from: gemsCatalystsRegistryOwner, log: true},
+    {from: gemsCatalystsRegistryAdmin, log: true},
```

Comment:
As mentioned above, if the admin was set to be the deployer inititaly, thsi could be executed by the deployer.
Otherwise, even that first step would require a multi sig tx